forward/forward.go

```diff
@@ -169,13 +172,6 @@ func (f *Forwarder) cloneRequest(req *http.Request, u *url.URL) (*http.Request,
 	outReq.URL.Scheme = "http"
 	// We need to make sure the host is defined in the URL (not the actual URI)
 	outReq.URL.Host = req.Host
-	// Make sure we define an opaque URL, so the URI is just the path
-	//
-	// Note that EscapedPath() always return the URL-encoded path, so we have
-	// no way to know if the raw URL in req is encoded or not.  Go HTTP client
-	// always encode URL, so do major browsers. That means it's totally safe for
-	// Lantern usage and possibly others.
-	outReq.URL.Opaque = req.URL.EscapedPath()
```

Comment:
This line of code is originally introduced in https://github.com/getlantern/http-proxy/commit/2fdfa3a1591e2756e2a297422d92e34a7f6e63fe to resolve https://github.com/getlantern/http-proxy/issues/28. I guess the purpose is to pass one of the Co-Advisor test. @uaalto Do you still remember the details?